apps/application-system/api/src/app/modules/application/files/utils/aws.ts

```diff
@@ -0,0 +1,55 @@
+import * as AWS from 'aws-sdk'
+import { environment } from '../../../../../environments'
```

Comment:
This feels super weird to see in a repo full of aliases. Should we create an alias if there isn't one for this?

Reply:
Totally agree, this is done this way in 35 places in the repo, so I am guessing there is some reason for why there is not an alias used for the environment. Feel that this is out of scope for this PR since this looks like the pattern that is being created for services in BE.

Reply:
Just noticed that this is also fixed in this PR: https://github.com/island-is/island.is/pull/2973/files/9241da83202f9ed51dce3cf43ac0cbaf89e309e2..37882b8f601e6332ae7caf3837df9922186a591e